css/default.css

```diff
@@ -143,6 +143,10 @@ img {
     line-height: 1.2em;
 }
 
+.idno-entry .h-card .p-name {
+    word-break: break-word;
```

Comment:
Is it possible this should be `word-wrap: break-word`? Based on MDN, I think so

https://developer.mozilla.org/en-US/docs/Web/CSS/word-wrap
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break